### PR TITLE
Text revision in the tests and doctrings for reward functions.

### PR DESCRIPTION
**Pull request by gasse**

### CODE REVIEW

File: python/src/ecole/core/reward.cpp [hunk 1]
```diff
@@ -63,35 +63,35 @@ void bind_submodule(py::module m) {
...
+		Always returns the value passed in constructor.
 	)");
 	constant  //
 		.def(py::init<Reward>(), py::arg("constant") = 0.)
 		.def_readonly("constant", &Constant::constant);
 	def_operators(constant);
-	def_reset(constant, "Do nothing.");
-	def_obtain_reward(constant, "Return the constant value.");
+	def_reset(constant, "Does nothing.");
+	def_obtain_reward(constant, "Returns the constant value.");
```
Comment (AntoinePrv) [inline python/src/ecole/core/reward.cpp:73]: PEP257

File: python/src/ecole/core/reward.cpp [hunk 2]
```diff
@@ -63,35 +63,35 @@ void bind_submodule(py::module m) {
...
+		An object of this class is returned by reward function operators to forward calls
+		to the reward function parameters of the operator.
 	)");
 	arithmetic  //
 		.def(py::init<py::object, py::list, py::str>())
 		.def("__repr__", &Arithmetic::toString);
 	def_operators(arithmetic);
 	def_reset(arithmetic, R"(
-		Reset the reward functions of the operator.
+		Resets the reward functions of the operator.
```
Comment (AntoinePrv) [inline python/src/ecole/core/reward.cpp:86]: PEP257

File: python/src/ecole/core/reward.cpp [hunk 3]
```diff
@@ -63,35 +63,35 @@ void bind_submodule(py::module m) {
...
 
-		Call ``reset`` on all reward functions parameters that were used to create this
+		Calls ``reset`` on all reward functions parameters that were used to create this
 		object.
 	)");
 	def_obtain_reward(arithmetic, R"(
-		Obtain the reward of result of the operator.
+		Obtains the reward of result of the operator.
```
Comment (AntoinePrv) [inline python/src/ecole/core/reward.cpp:92]: PEP257

File: python/src/ecole/core/reward.cpp [hunk 4]
```diff
@@ -111,22 +111,22 @@ void bind_submodule(py::module m) {
 	auto isdone = py::class_<IsDone>(m, "IsDone", "Single reward on terminal states.");
 	isdone.def(py::init<>());
 	def_operators(isdone);
-	def_reset(isdone, "Do nothing.");
-	def_obtain_reward(isdone, "Return 1 if the episode is on a terminal state, 0 otherwise.");
+	def_reset(isdone, "Does nothing.");
+	def_obtain_reward(isdone, "Returns 1 if the episode is on a terminal state, 0 otherwise.");
```
Comment (AntoinePrv) [inline python/src/ecole/core/reward.cpp:115]: PEP257

File: python/src/ecole/core/reward.cpp [hunk 5]
```diff
@@ -111,22 +111,22 @@ void bind_submodule(py::module m) {
...
 		LP Iteration difference.
 
-		The reward is defined as the number of iterations done solving the Linear Program
+		The reward is defined as the number of iterations spent in solving the Linear Programs
 		associated with the problem since the previous state.
 	)");
 	lpiterations.def(py::init<>());
 	def_operators(lpiterations);
-	def_reset(lpiterations, "Reset the internal LP iterations count.");
+	def_reset(lpiterations, "Resets the internal LP iterations count.");
```
Comment (AntoinePrv) [inline python/src/ecole/core/reward.cpp:125]: PEP257

File: python/src/ecole/core/reward.cpp [hunk 6]
```diff
@@ -111,22 +111,22 @@ void bind_submodule(py::module m) {
...
 	def_obtain_reward(lpiterations, R"(
-		Update the internal LP iteration count and return the difference.
+		Updates the internal LP iteration count and return the difference.
```
Comment (AntoinePrv) [inline python/src/ecole/core/reward.cpp:127]: PEP257

**Comment by gasse** [comment]
Oh I wasn't aware of that standard. I was always told it was better to employ a descriptive style in method descriptions. I guess the PEP guys have thought this through.

**Comment by AntoinePrv** [comment]
I don't really care for or against it, but it the convention in the rest of the project, so why change it.

**Comment by gasse** [comment]
Agreed. I don't really care either, but let's be consistent. Thanks !

**Pull request merged**